pt/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Resumo de Aprendizagem Profunda
 
 <br>
 
 **2. Neural Networks**
 
-&#10230;
+&#10230; Redes Neurais
 
 <br>
 
 **3. Neural networks are a class of models that are built with layers. Commonly used types of neural networks include convolutional and recurrent neural networks.**
 
-&#10230;
+&#10230; Redes neurais são uma classe de modelos que são construídos com camadas. Os tipos de redes neurais comumente utilizadas incluem redes neurais convolucionais e recorrentes.
 
 <br>
 
 **4. Architecture ― The vocabulary around neural networks architectures is described in the figure below:**
 
-&#10230;
+&#10230; Arquitetura - O vocabulário em torno das arquiteruras de redes neurais é descrito na figura abaixo:
 
 <br>
 
 **5. [Input layer, hidden layer, output layer]**
 
-&#10230;
+&#10230; [Camada de entrada, camada escondida, camada de saída]
 
 <br>
 
 **6. By noting i the ith layer of the network and j the jth hidden unit of the layer, we have:**
 
-&#10230;
+&#10230; Note que i é a i-ésima camada da rede e j a j-ésima unidade escondida da camada, nós temos:
 
 <br>
 
 **7. where we note w, b, z the weight, bias and output respectively.**
 
-&#10230;
+&#10230; onde notamos w, b, z, o peso, o viés e a saída respectivamente. 
 
 <br>
 
 **8. Activation function ― Activation functions are used at the end of a hidden unit to introduce non-linear complexities to the model. Here are the most common ones:**
 
-&#10230;
+&#10230; Função de ativação - Funções de ativação são usadas no fim de uma unidade escondida para introduzir complexidades não lineares ao modelo. Aqui estão as mais comuns:
 
 <br>
 
 **9. [Sigmoid, Tanh, ReLU, Leaky ReLU]**
 
-&#10230;
+&#10230; [Sigmoide, Tanh (tangente hiperbólica), ReLu (unidade linear retificada), Leaky ReLU]
```

Comment:
I believe that `Sigmoide` has an accent

Reply:
This one was a doubt that I have. However, I think this word was changed by the orthographic agreement. I found this info at link: https://portuguesaletra.com/acordo-ortografico/sigmoide-ou-sigmoide-ao/

Reply:
Ok! Didn't know about this. I'll change it on mine.